src/WordSearch.java

```diff
@@ -20,26 +25,34 @@ public static void main(String[] args) {
     /*
     Checking File Extension of the filePath whether '.txt' or '.json' file.
     And Calling processFile method for searching the file in system.
+    If format of file is not supported, Creating an object for WordSearchHelper and calling a method dataBaseStorage from that class.
+    Storing the results in DataBase.
      */
-    public static void fileExtensionCheck(String filePath, String keyWordToSearch) {
+    public static void fileExtensionCheck(String filePath, String keyWordToSearch) throws SQLException {
         if (filePath.endsWith(".txt") || filePath.endsWith(".json")) {
```

Comment:
better to move .txt and .json moved into contant variable

Reply:
Made changes as  per suggestion.